### PR TITLE
Revert merging develop for bionic/stretch fixes

### PR DESCRIPTION
**Pull request by mitschabaude**
This reverts commit 4afc7b55b5a8549902bedecdcc3cc7fbd828e06d, reversing
changes made to 409d605d92fe037a976e481abed503d4e9844045.

### CODE REVIEW

**Comment by mitschabaude** [comment]
!ci-build-me

**Review by Trivo25 — APPROVED** [review]

**Review by lk86 — APPROVED** [review]

File: frontend/ci-build-me/src/index.js [hunk 1]
```diff
@@ -76,11 +77,10 @@ const handler = async (event, req) => {
     ) {
       // TODO #7711: Actually look at @MinaProtocol/stakeholder-reviewers team instead of hardcoding the users here
       if (
+        req.body.sender.login == "es92" ||
         req.body.sender.login == "aneesharaines" ||
         req.body.sender.login == "bkase" ||
-        req.body.sender.login == "deepthiskumar" ||
```
[the remark this answers is not in the document]
Comment (deepthiskumar) [reply frontend/ci-build-me/src/index.js:81]: I think it is sufficient to have it in `develop`? If not, we can make a new PR to make only this change

**Review by ghost-not-in-the-shell — APPROVED** [review]

**Review by fabrizio-m — APPROVED** [review]

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by mitschabaude** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Pull request merged**